app/assets/src/components/views/SampleViewV2/SampleViewV2.jsx

```diff
@@ -965,7 +965,10 @@ export default class SampleViewV2 extends React.Component {
       selectedOptions,
       view,
     } = this.state;
-    if (reportMetadata.pipelineRunStatus === "COMPLETE") {
+    if (
+      reportMetadata.pipelineRunStatus === "COMPLETE" &&
+      !reportMetadata.hasErrors
```

Comment:
We don't want to do this because on experimental errors. In that case we need to show the report.

Reply:
Replaced with `pipelineRunReportAvailable`